release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -593,9 +592,80 @@ For more information about the `cluster-compare` plugin, see xref:../scalability
+==== Multiple vCenter support for vSphere CSI is generally available
+{product-title} 4.17 introduced the ability to deploy {product-title} across multiple vSphere clusters (vCenters) as a Technology Preview feature. In {product-title} 4.18, Multiple vCenter support is now generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-multi-vcenter-support-overview_persistent-storage-csi-vsphere[Multiple vCenter support for vSphere CSI] and xref:../installing/installing_vsphere/installation-config-parameters-vsphere.adoc[Installation configuration parameters for vSphere].
+
+[id="ocp-4-18-release-notes-storage-always-honor-reclaim-policy_{context}"]
+==== Always honored persistent volume reclaim policy (Technical Preview)
+Prior to {product-title} 4.18, the persistent volume (PV) reclaim policy was not always applied.
+
+For a bound PV and persistent volume claim (PVC) pair, the ordering of PV-PVC deletion determined whether the PV delete reclaim policy was applied or not. The PV applied the reclaim policy if the PVC was deleted prior to deleting the PV. However, if the PV was deleted prior to deleting the PVC, then the reclaim policy was not applied. As a result of that behavior, the associated storage asset in the external infrastructure was not removed.
+
+With {product-title} 4.18, the PV reclaim policy is consistently always applied. This feature has Technical Preview status.
+
+For more information, see xref:../storage/understanding-persistent-storage.adoc#reclaiming_understanding-persistent-storage[Reclaim policy for persistent volumes].
+
+[id="ocp-4-18-release-notes-storage-lso-cleanup_{context}"]
+==== Improved ability to easily remove LVs or LVSs for LSO is generally available
+For the Local Storage Operator (LSO), {product-title} 4.18 improves the ability to remove Local Volumes (LVs) and Local Volume Sets (LVSs) by automatically removing artifacts, thus reducing the number of steps required.
+
+For more information, see xref:../storage/persistent_storage/persistent_storage_local/persistent-storage-local.adoc#local-removing-device_persistent-storage-local[Removing a local volume or local volume set].
+
+[id="ocp-4-18-release-notes-storage-volume-group-snapshots_{context}"]
+==== CSI volume group snapshots (Technology Preview)
+{product-title} 4.18 introduces Container Storage Interface (CSI) volume group snapshots as a Technology Preview feature. This feature needs to be supported by the CSI driver. CSI volume group snapshots use a label selector to group multiple persistent volume claims (PVCs) for snapshotting. A volume group snapshot represents copies from multiple volumes that are taken at the same point-in-time. This can be useful for applications that contain multiple volumes. 
+
+{product-title} supports volume group snapshots by default. {rh-storage} supports volume group snapshots.
```

Comment:
I think we can remove this line because it's not enabled by default

Reply:
Done.